pkg/primitives/zdb.go

```diff
@@ -36,6 +37,22 @@ type ZDB = zos.ZDB
 
 type tZDBContainer pkg.Container
 
+type safeError struct {
+	error
+}
+
+func newSafeError(err error) error {
+	if err == nil {
+		return nil
+	}
+	return &safeError{err}
+}
+func (se *safeError) Error() string {
+	return regexp.
+		MustCompile(`([0-9a-f]{8})\b-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\b[0-9a-f]{12}`).
```

Comment:
You can compile the regex one time and leave it as a private global variable in this package. Then just use it as 
```go
re.ReplaceAllString(...)
```